Leanplum-SDK/Classes/Internal/Leanplum.m

```diff
@@ -1133,7 +1134,9 @@ + (void)startWithUserId:(NSString *)userId
     LP_END_TRY
     
     LP_TRY
-    [LPUtils initExceptionHandling];
+    if ([[LPFeatureFlagManager sharedManager] isFeatureFlagEnabled:LP_FEATURE_FLAG_CRASH_MONITORING]) {
```

Comment:
Same as Android. A simple test case with mock feature flag and whether this exection api gets called would be great.